### ISSUE TITLE
Ability for author to delete

### ISSUE THREAD

**Issue by pcrossnz** [fergusfrl/RiverWiki#31]
Will be good to have the ability of the author of a description (or a superuser) to delete posts. For example, my crappy Avon river one now...

**Comment by fergusfrl** [comment]
I agree, I might create an admin login which can do something like that.
For now we can delete directly from the database

**Comment by pcrossnz** [comment]
Sounds like a plan.
